doc/overview.rst

```diff
@@ -29,13 +29,30 @@ rosdep gets its data from.
 ``rosdep update`` reads through this sources list to initialize your
 local database.
 
+You can override the location by setting environment variable
+``ROSDEP_SOURCE_PATH``. The custom path has to exist prior to calling
+``rosdep init``, otherwise the default one will be used.
+
+Please note that when using ``sudo``, environment
+variables from the user are not passed to the command. To specify the variable
+for initializing the database, call::
+
+    sudo mkdir -p /usr/rosdep.sources
+    sudo env ROSDEP_SOURCE_PATH=/usr/rosdep.sources rosdep init
+
 Updating rosdep
 ---------------
 
 You can update your rosdep database by running::
 
     rosdep update
 
+If you have specified a custom ``ROSDEP_SOURCE_PATH``, do not forget to set it
+also for this command.
+
+Default location of the local rosdep database is in ``$HOME/.ros/rosdep``.
+To change it, set environment variable ``ROSDEP_CACHE_PATH``, or pass
+command-line arguments ``--sources-cache-dir`` and ``--meta-cache-dir``.
```

Comment:
Which value is used in the event both `ROSDEP_CACHE_PATH` and `--sources-cache-dir` are used? I would normally expect command line arguments to override environment variables but this PR has 'deprecate `--sources-cache-dir` in the title.

Can you please document what happens in this case?

Reply:
Documented, CLI options win.